.github/workflows/trusted-ci-linux-on-main-pr.disabled

```diff
@@ -43,12 +43,12 @@ jobs:
           sudo apt-get install -y 
           sudo apt-get remove -y llvm-12
           sudo apt-get install -y curl pkg-config findutils  wget
-          sudo apt install -y clang-11 cmake clang-format-11 clang-tidy-11 
-          sudo apt-get install -y llvm-11 lldb-11 llvm-11-dev libllvm11 llvm-11-runtime opt 
+          sudo apt install -y clang-13 cmake clang-format-13 clang-tidy-13 
+          sudo apt-get install -y llvm-13 lldb-13 llvm-13-dev libllvm11 llvm-13-runtime opt 
           sudo apt-get install -y nodejs npm
           sudo apt install -y python3 python3-pip 
           sudo apt-get remove -y llvm-10
-          sudo ln -s /usr/lib/llvm-11/bin/opt /usr/bin/opt
+          sudo ln -s /usr/lib/llvm-13/bin/opt /usr/bin/opt
```

Comment:
(nit)
I have an impression that there is a chance of code duplication between multiple .yml files. If that's the case then consider extracting a common part into a separate .yml file and invoking the common part from multiple .yml files
(when and if you have time).

Reply:
This is a good suggestion - I was not aware that you could do this with Github actions. I will do this change once `qiskit-qir` is ready to be reinstated in the test pipelines.